crates/bevy_window/src/event.rs

```diff
@@ -58,10 +58,18 @@ pub struct WindowCloseRequested {
 pub struct WindowClosed {
     pub id: WindowId,
 }
-/// An event that is sent whenenver the user's cursor moves.
+/// An event reporting that the mouse cursor has moved on a window.
+///
+/// The event is sent only if the cursor is over one of the application's windows.
```

Comment:
Maybe this statement is redundant? (see first doc line)

Reply:
I think this is a useful clarification.